test/extended/util/framework.go

```diff
@@ -40,13 +40,79 @@ var TestContext e2e.TestContextType
 
 const pvPrefix = "pv-"
 
+//FixOpenShiftNamespaceImageStreams is a temporary workaround for the intermittent
+// issue seen in extended tests where *something* is deleteing the pre-loaded, languange
+// imagestreams from the OpenShift namespace
+func FixOpenShiftNamespaceImageStreams(oc *CLI) {
```

Comment:
Can we not add logging to the imagestream deletion rest handler or enable auditing to catch this, instead of this workaround?